extensions/BugModal/web/bug_modal.css

```diff
@@ -729,6 +729,15 @@ div.comment-text blockquote {
     padding: 0.5em 10px;
 }
 
+
+div.comment-text blockquote p:first-child {
```

Comment:
Just giving my two cents here: I think removing the `p` specificity would be good. Some blockquotes start/end with H1, or other tags and those would still have the big margins. E.g. `div.comment-text blockquote :first-child`.
This change successfully reduces top/bottom margins on the blockquotes, an oversight on my part!